test/integration/scheduler/framework_test.go

```diff
@@ -1564,19 +1519,13 @@ func TestPreemptWithPermitPlugin(t *testing.T) {
 	cleanupPods(cs, t, []*v1.Pod{waitingPod, preemptorPod})
 }
 
-func initTestContextForScorePlugin(t *testing.T, plugins *schedulerconfig.Plugins, registry framework.Registry) (*testContext, *clientset.Clientset) {
-	// Set empty plugin config for testing
-	emptyPluginConfig := []schedulerconfig.PluginConfig{}
-
-	// Create the master and the scheduler with the test plugin set.
-	context := initTestSchedulerWithOptions(t,
-		initTestMaster(t, "score-plugin", nil),
-		false, nil, registry, plugins, emptyPluginConfig, time.Second)
-
-	cs := context.clientSet
-	_, err := createNodes(cs, "test-node", nil, 10)
-	if err != nil {
-		t.Fatalf("Cannot create nodes: %v", err)
+func initTestSchedulerForFrameworkTest(t *testing.T, context *testContext, nodeCount int, opts ...scheduler.Option) (*testContext, *clientset.Clientset) {
```

Comment:
There seems to be no need to return the clientset as it is part of the context.

Tangential: Let's rename testContext to something else: `testResources`? But probably another PR.